typescript

```diff
@@ -0,0 +1,135 @@
+Script started on 2018-03-06 11:15:57+0530
+[1m[7m#[27m[1m[0m                                                                               
 
]2;root@localhost: /home/ihackpy/Panda/pandas-dir]1;..da/pandas-dir
[0m[27m[24m[J[01;32m➜  [36mpandas-dir[00m [01;34mgit:([31mdocstringhead[34m) [33m✗[00m [K[?1h=[?2004hlls[?1l>[?2004l

+]2;ls --color=tty]1;lsappveyor.yml  [0m[01;34mconda.recipe[0m  [01;34mpandas[0m            setup.cfg        [01;32mtest.sh[0m

```

Comment:
Can you remove this file again?

Reply:
Sorry I don't know HOw this file added sorry for the inconvenience